compiler/data/class-data.h

```diff
@@ -64,6 +64,7 @@ class ClassData : public Lockable {
   bool has_custom_constructor{false};
   bool is_serializable{false};
   bool has_job_shared_memory_piece{false};
+  bool is_readonly{false};
```

Comment:
Do we really need it?
is this property not expressed through `is_immutable`?

Reply:
I agree, it's better to use `is_immutable `.